web-client/src/ducks/timeline/functions.ts

```diff
@@ -1,15 +1,114 @@
 import { firestore } from 'src/firebase';
-import { TimelineItemFirestoreConverter } from 'src/models/requests/timeline';
+// eslint-disable-next-line no-unused-vars
+import { IOffer, Offer } from 'src/models/offers';
+import { IRequest, Request, RequestFirestoreConverter, RequestRefType } from 'src/models/requests';
+import { IRequestWithOffersAndTimelineItems, RequestWithOffersAndTimeline } from 'src/models/requests/RequestWithOffersAndTimeline';
+// eslint-disable-next-line no-unused-vars
+import { ITimelineItem, TimelineItemAction, TimelineItemFirestoreConverter } from 'src/models/requests/timeline';
 
-import { IgetTimeline } from './types';
+const getRequest = async (requestRef: RequestRefType): Promise<IRequest> => {
+  console.log('abcd');
+  let requestData;
+    await firestore
+      .collection('requests')
+      .doc('1EQlpGu0QQMBji6MEPZ7')
+      .withConverter(RequestFirestoreConverter)
+      .get()
+      .then(snapshot => {
+          console.log('xxx',snapshot);
+          requestData = {
+          id: snapshot.id,
+          _requestRef: snapshot.ref,
+          // TODO:(es) Convert to request here?
+          ...snapshot.data(),
+        };
+      });
 
-export const observeTimelinesFunc = (
-  nextValue: Function,
-  payload: IgetTimeline,
-): firebase.Unsubscribe =>
+  console.log('xxxx', requestData, 'xxxz', requestRef);
```

Comment:
requestData for some reason is not a request and has onl underscores in it.  I tried various changes with no luck.